src/pudl/glue/epacems_unitid_eia_plant_crosswalk.py

```diff
@@ -0,0 +1,92 @@
+"""Extract, clean, and normalize the EPA-EIA crosswalk.
+
+This module defines functions that read the raw EPA-EIA crosswalk file, clean
+up the column names, and separate it into three distinctive normalize tables
+for integration in the database.
+
+The crosswalk file was a joint effort on behalf on EPA and EIA and is published on the
+EPA's github account at www.github.com/USEPA".
+"""
+import logging
+
+import pandas as pd
+
+import pudl
+from pudl.metadata.fields import apply_pudl_dtypes
+from pudl.workspace.datastore import Datastore
+
+logger = logging.getLogger(__name__)
+
+
+def extract(ds: Datastore) -> pd.DataFrame:
+    """Extract the EPACEMS-EIA Crosswalk from the Datastore."""
+    with ds.get_zipfile_resource(
+        "epacems_unitid_eia_plant_crosswalk",
+        name="epacems_unitid_eia_plant_crosswalk.zip",
+    ).open("camd-eia-crosswalk-master/epa_eia_crosswalk.csv") as f:
+        return pd.read_csv(f)
+
+
+def transform(epa_eia_crosswalk: pd.DataFrame) -> pd.DataFrame:
+    """Clean up the EPACEMS-EIA Crosswalk file."""
+    logger.info("Cleaning up the epacems-eia crosswalk")
+    column_rename = {
+        "camd_unit_id": "unit_id_epa",
+        "camd_plant_id": "plant_id_epa",
+        "eia_plant_name": "plant_name_eia",
+        "eia_plant_id": "plant_id_eia",
+        "eia_generator_id": "generator_id_eia",
```

Comment:
Should this really be `generator_id_eia`? We just call it `generator_id` in PUDL (perhaps inadvisedly)

Reply:
I thought about that...It really should be `generator_id_eia` imo, but the rest of the data uses `generator_id`. For now it's probably smart to fix those things separately. I.e.: use `generator_id` and make an issue about changing `generator_id` to `generator_id_eia` later. It's just confusing to have an unlabeled field in a crosswalk table. Wouldn't want to assume it was epa!

Reply:
The central challenge of the EPA crosswalk is to wrangle IDs from different sources, so I'd vote for being as explicit as possible with the naming convention, like Austen did here. I think we already have a collision in the CEMS data - PUDL calls it `plant_id_eia` but the CEMS ID fields are really from CAMD, which don't always match.